### ISSUE TITLE
Grant me repository admin access

### ISSUE THREAD

**Issue by Exactol** [ruarai/CompilePal#193]
Hello @ruarai, can you grant me admin access to the repository? The link in the project description no longer works and I would like to fix it.

**Comment by ruarai** [comment]
Hey, thanks very much for taking care of this project in my absence.

It doesn't look like I can add another admin to the repository as I don't have a paid github account. But I think I can transfer the repo as a whole to your account which I'd be happy to do. How does that sound?

**Comment by maxdup** [comment]
Transferring the repo would turn ruarai/CompilePal into Exactol/CompilePal. 

This breaks all urls to https://github.com/ruarai/CompilePal

I'm not opposed to Exactol/CompilePal but instead I think it should be a fork where active development can continue.
This preserves ruarai/CompilePal and its links for historical purposes. We Could then update the ruarai/CompilePal readme to redirect users to Exactol/CompilePal

(This is suggestion/personal preference)

**Comment by Exactol** [comment]
I agree that breaking the URLs is not ideal, I'm fine with keeping development on this repository. Can you edit the description to remove the broken URL? Also, I got into the GitHub Sponsors beta recently, would it be possible for you to enable the Sponsorships feature in the repository settings?
